### ISSUE TITLE
jdk_jdi_jdk8 fails because of TestClassLoaderLeak.java failed when GC cycles are not happening 

### ISSUE THREAD

**Issue by ChangLu0607** [adoptium/aqa-tests#3572]
**Describe the bug**
TestClassLoaderLeak.java faild with JDK8 hotspot

**To Reproduce**
make _jdk_jdi_jdk8_0

**Expected behavior**
all test cases pass

**Screenshots**
![image](https://user-images.githubusercontent.com/38229679/163148590-28280856-930b-4e2f-b906-e569aafbd35c.png)

**Additional context**
realated issue in latest version of openJDK: https://bugs.openjdk.java.net/browse/JDK-8274563
https://github.com/openjdk/jdk/commit/47bfc8aa9367ff852ea5d901f1fa3c6ef316913e

**Comment by sophia-guo** [comment]
Test passed in adoptium recent release build https://ci.adoptopenjdk.net/job/Test_openjdk8_hs_extended.openjdk_x86-64_linux_testList_1/37/testReport/jdk_jfr_event_oldobject_TestClassLoaderLeak/java/.

@ChangLu0607 are you using the recent jdk8 release branch?

**Comment by ChangLu0607** [comment]
@sophia-guo  I believe that you are looking at an other failed test case BiSheng JDK also pass this test case in adoptium. So I believe it may be a environment setting problem that makes the test case failed intermittently. 

> Test did fail in recent run in adoptium with bisheng jdk.
> 
> https://ci.adoptopenjdk.net/view/Test_openjdk/job/Test_openjdk8_bisheng_sanity.openjdk_x86-64_linux/124/testReport/com_sun_jdi_RedefineCrossEvent/java/RedefineCrossEvent

**Comment by sophia-guo** [comment]
Yes, I was looking at the other failed test case for another issue. The test passed in adoptium bisheng build. https://ci.adoptopenjdk.net/job/Test_openjdk8_bisheng_extended.openjdk_x86-64_linux_testList_1/28/testReport/jdk_jfr_event_oldobject_TestClassLoaderLeak/java/

**Comment by sendaoYan** [comment]
I think it's a [testcae bug](https://github.com/openjdk/jdk/commit/47bfc8aa9367ff852ea5d901f1fa3c6ef316913e) for this fail.

Should we disable this testcase by add this testcase to openjdk/excludes/ProblemList_openjdk8.txt ?
